subprojects/model-core/src/main/java/org/gradle/internal/reflect/JavaPropertyReflectionUtil.java

```diff
@@ -66,7 +66,7 @@ public static <T, F> PropertyAccessor<T, F> readableProperty(T target, Class<F>
     }
 
     @Nullable
-    private static Method findGetterMethod(Class<?> target, String property) {
+    public static Method findGetterMethod(Class<?> target, String property) {
```

Comment:
We should keep this private and instead use `JavaPropertyReflectionUtil.readableProperty()`

Reply:
The reason I chose to do this, instead of what you are suggesting, is that the `readableProperty` method needs a return type to be provided as input, which I don't know/have.

Plus it creates an instance of a `PropertyAccessor` as the return type, which I also don't need.

Btw. these `readableProperty` methods have no usages in the whole project (other than their own tests), so we might even delete them.

Reply:
Right, I misse the return type parameter.
:+1: